src/ape_compile/_cli.py

```diff
@@ -42,6 +42,11 @@ def _include_dependencies_callback(ctx, param, value):
     help="Also compile dependencies",
     callback=_include_dependencies_callback,
 )
+@click.option(
+    "--exclude-compiler",
+    type=click.Choice(["solidity", "vyper"], case_sensitive=False),
```

Comment:
This shouldn't be a click.Choice. There are more compilers besides `solidity` and `vyper`. With Ape's plugin system, there isn't a fixed number of compilers.

A normal `str` type is just find.